statsd/statsd.go

```diff
@@ -94,8 +100,9 @@ const (
 )
 
 const (
-	WriterNameUDP string = "udp"
-	WriterNameUDS string = "uds"
+	WriterNameUDP     string = "udp"
+	WriterNameUDS     string = "uds"
+	WriterWindowsPipe string = "pipe"
```

Comment:
Same about those, could we make them private

Reply:
We could, but that would be a breaking change. Technically it's not ok to do that in open-source projects but if you want me too I can do it.

Reply:
Indeed missed that. Let's keep those public.